contributing/code/conventions.rst

```diff
@@ -100,40 +100,38 @@ A feature is marked as deprecated by adding a ``@deprecated`` PHPDoc to
 relevant classes, methods, properties, ...::
 
     /**
-     * @deprecated since Symfony 2.8.
+     * @deprecated since Symfony 5.1.
      */
 
 The deprecation message must indicate the version in which the feature was deprecated,
 and whenever possible, how it was replaced::
 
     /**
-     * @deprecated since Symfony 2.8, use Replacement instead.
+     * @deprecated since Symfony 5.1, use Replacement instead.
      */
 
 When the replacement is in another namespace than the deprecated class, its FQCN must be used::
 
     /**
-     * @deprecated since Symfony 2.8, use A\B\Replacement instead.
+     * @deprecated since Symfony 5.1, use A\B\Replacement instead.
      */
 
 A PHP ``E_USER_DEPRECATED`` error must also be triggered to help people with the migration::
 
-    @trigger_error(sprintf('The "%s" class is deprecated since Symfony 2.8, use "%s" instead.', Deprecated::class, Replacement::class), E_USER_DEPRECATED);
+    trigger_deprecation('vendor-name/package-name', '5.1', 'The "%s" class is deprecated since Symfony 5.1, use "%s" instead.', Deprecated::class, Replacement::class);
```

Comment:
`since Symfony 5.1` should be removed from the message

Reply:
This was fixed while merging (a2846b9f04b371bc3f1f11f87075b2c686799861)